plugins/modules/cloudwatch_metric_alarm_info.py

```diff
@@ -0,0 +1,203 @@
+#!/usr/bin/python
+# This file is part of Ansible
+# GNU General Public License v3.0+ (see COPYING or https://wwww.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+module: cloudwatch_metric_alarm_info
+version_added: 5.0.0
+short_description: Gather information about the alarms for the specified metric
+description:
+  - Retrieves the alarms for the specified metric.
+author:
+  - Mandar Vijay Kulkarni (@mandar242)
+options:
```

Comment:
We should support the option specified here https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/cloudwatch.html#CloudWatch.Client.describe_alarms